src/ui.tsx

```diff
@@ -804,9 +804,21 @@ function PlotButtons(
     driver.pause();
   }
   function resume() {
+    if (state.planOptions.pauseAfter && state.planOptions.pauseAfter>0) {
+      setTimeout(() => {
+        console.log('Plot paused again after ', state.planOptions.pauseAfter, ' seconds');
+        driver.pause();
+      }, state.planOptions.pauseAfter * 1000);
+    }
     driver.resume();
   }
   function plot(plan: Plan) {
+    if (state.planOptions.pauseAfter && state.planOptions.pauseAfter>0) {
+      setTimeout(() => {
+        console.log('Plot paused after ', state.planOptions.pauseAfter, ' seconds');
+        driver.pause();
+      }, state.planOptions.pauseAfter * 1000);
```

Comment:
This isn't the right way to go about implementing this. For example, if you pause the plot manually before the timer fires, then resume, this timer will still be running. And worse—there will actually be _two_ timers running, since resuming creates another timer. The timer also isn't canceled after canceling a plot, so it could even fire on a totally different plot.

I'd accept some similar feature to this if it were integrated into the driver code. i.e. after each action, it could check the elapsed time against the pause duration. Also, it might make sense to have it only count time while the pen is down? Or possibly to count "pen-meters" rather than seconds?

Reply:
Alrighty, glad this helped you out and it was easy to hack in! I'll close this PR for now.

Reply:
maybe something like what was suggested in #69 could work to indicate breakpoints?  would be based on geometry not time, but would work with batches etc.

Reply:
Maybe yes, but it is then really far away from my original need, a simple solution that every X seconds do a pause to do anything needed on a pen (shake, change, etc), without any need to pre-process SVG file. I'm gonna stick with that simple use case for now, fixing the duplicate timer problem.